src/js/components/OptionalPortsAndServiceDiscoveryComponent.jsx

```diff
@@ -32,9 +32,14 @@ function determinePortDefinitionsType(fields) {
 
 function isTooComplexStructure(fields) {
   return fields.portDefinitions != null &&
-      fields.portDefinitions.some(portDefinition =>
-        portDefinition.labels != null &&
-      !!Object.keys(portDefinition.labels).length
+      fields.portDefinitions.some(portDefinition => {
+        var definition = Object.assign({}, portDefinition);
+        if (definition.labels != null) {
+          delete definition.labels["VIP_0"];
```

Comment:
Actually, there is no modified data returned or modified outside of the function scope.
So this is no problem, it's just for detection.
The data is cloned at line 36.